src/components/users/user-signup/ko/runtime/user-signup.ts

```diff
@@ -84,20 +92,15 @@ export class UserSignup {
         }
 
         const settings = {
-            // TODO: Registration terms could be rendered at publish time
-            userRegistrationTerms: "Test userRegistrationTerms!!!",
-            userRegistrationTermsEnabled: false,
-            userRegistrationTermsConsentRequired: false
+            userRegistrationTerms: this.termsOfUse(),
+            userRegistrationTermsEnabled: this.termsEnabled(),
+            userRegistrationTermsConsentRequired: this.isConsentRequired()
         };
-
         this.tenantSettings = settings as TenantSettings;
```

Comment:
I do not see any good reasons to keep this **tenantSettings **
do you really need it?

Reply:
sure i will delete it